presto-geospatial/src/test/java/io/prestosql/plugin/geospatial/BenchmarkGeometryAggregations.java

```diff
@@ -67,7 +67,7 @@ public void setUp()
         {
```

Comment:
Please add plenty of tests to `BaseConnectorTest`.

Things to cover:
 - whether output column aliases for remain their case
 - CREATE TABLE with case sensitive names
 - CREATE TABLE with columns that differs only with casing, do select, insert, update, delete from from that table
 - The similar like above with columns, but for schemas and catalogs.